src/js/functions/sort/sortTodos.js

```diff
@@ -0,0 +1,10 @@
+import currentStates from '../currentStates';
+import projectTodos from '../project/projectTodos';
+
+const sortTodos = () => {
+  const sortSelectValue = document.querySelector('#sortSelect').value;
+  currentStates.sortSelect = sortSelectValue;
+  projectTodos[currentStates.filterSelect][sortSelectValue](currentStates.projectState);
+};
+
+export default sortTodos;
```

Comment:
Let `sortTodos` accept a parameter only, sort the parameter and return a sorted parameter without mutating the existing one. Let's not miss up DOM manipulation with this app logic.

Reply:
This code is for the DOM manipulation for sorting todos. I'll ensure to move the file to `dom` folders as necessary.